docs/development-tips.md

```diff
@@ -34,6 +34,11 @@ Feel free to [install a plugin](https://editorconfig.org/#download) for your edi
 
 Some (but not all) JavaScript and CSS files can be rebuilt automatically when they are changed by using `yarn watch`.
 
+## Outputting browser support metrics
+
+Within the root directory, run `npx browserslist` to output the set of browser
+targets given to `esbuild` (for JS) and `autoprefixer` (CSS) transpiling.
+
```

Comment:
> We could do "browserslist": "defaults", in package.json if we want to, but I think as written it's a little clearer.

Can you add this note (and maybe a link to the browserslist docs) to this section for folks who are unfamiliar with the browserslist syntax.

Reply:
https://github.com/cfpb/consumerfinance.gov/pull/7001/commits/6e095a678f267a0ee60774362d0793caa839f74b